### ISSUE TITLE
Responsive UI

### ISSUE THREAD

**Issue by sm2x** [openboxes/openboxes#437]
is there a future plan to support UI for mobile or is there an existing android/ios App already?
At least the dashboard, analytics, and reporting UI, managers, supervisors who are monitoring activities prefer to view these information on their mobile devices.

**Comment by jmiranda** [comment]
Yes, the plan is to implement new features with React and Bootstrap and slowly migrate the entire UI to use at least Bootstrap. See Roadmap (https://openboxes.com/roadmap). There's no android/ios app yet, but v0.8.0 will include a REST API that will allow someone to start work on a mobile app.

I agree with the idea of starting with certain features (i.e. dashboard / reporting). I would really like to see some of the warehouse activities (cycle counts, picking, putaway) implemented in a mobile version as well. I'll give that some thought and see if we can make that happen sooner rather than later.

As for the Dashboard, I have never felt like it was all that useful. If you have some concrete ideas for widgets (i.e. indicators, graphs, tables) that you'd like to see, please add some tickets to help us make it more useful.

**Comment by tech4kstudios** [comment]
Hey, what are the requirements for the new UI changes? I might start contributing where I can. Especially for CSS stuff. Also, what's the link to your Slack discussions?

**Comment by jmiranda** [comment]
We don't have specific requirements at the moment, but we're developing a style guide as we go. Will try to share that when I get a chance. As you discovered, Slack signup is available through http://slack-signup.openboxes.com/ (note: be patient ... the signup app is running on a free Heroku instance so it goes to sleep sometimes).

**Comment by ewaterman** [comment]
I'm going to close this out since the conversation has slowed over the years and I feel the subject is too broad to be useful currently. We do have some mobile initiatives that are being worked on but nothing that is currently tracked and documented via issues in this repo.

Once we get more structure to the mobile app, we can create issues for the individual features that we want to add.

**Comment by jmiranda** [comment]
The mobile app is being actively developed in the openboxes-mobile repo. Current requirements are in a private Jira instance (we are not planning to make that public) but you can watch development via PRs on the aforementioned repo.

https://github.com/openboxes/openboxes-mobile
